### DISCUSSION TITLE
How can I show category wise recent posts in my homepage?

### DISCUSSION THREAD

**Discussion by thecoducer** [nunocoracao/blowfish#636] [Q&A]
I have several categories like Technical, Travel, and Problem Solving for which I want to show at least 5 recent posts from each category on the homepage. Currently, the homepage shows recent posts without any category-wise grouping.

Here is my site: https://thecoducer.pages.dev/
Repo: https://github.com/thecoducer/blog

The homepage (https://thecoducer.pages.dev/) should show the posts category-wise whereas this page (https://thecoducer.pages.dev/articles/) should continue to display all the posts.

Can someone please guide me on how to do that?

Structure of my content folder:
```
.
├── about
├── categories
│   ├── events
│   ├── opinion
│   ├── technical
│   └── travel
├── jupyter-notebooks
├── posts
│   ├── 2017
│   │   ├── 04
│   │   ├── 07
│   │   ├── 08
│   │   └── 09
│   ├── 2018
│   │   ├── 01
│   │   ├── 02
│   │   └── 10
│   ├── 2019
│   │   ├── 01
│   │   ├── 04
│   │   ├── 06
│   │   ├── 07
│   │   └── 09
│   ├── 2020
│   │   ├── 08
│   │   └── 12
│   ├── 2021
│   │   ├── 01
│   │   └── 04
│   ├── 2022
│   │   ├── 01
│   │   ├── 02
│   │   ├── 03
│   │   └── 12
│   └── 2023
│       └── 03
├── problem-solving
│   ├── 2017
│   │   ├── 08
│   │   └── 09
│   ├── 2018
│   │   ├── 09
│   │   └── 10
│   ├── 2019
```

**Comment by nunocoracao** [comment]
Not available at the time. Please open a new ticket if you think it would be a good addition.
